commons-vfs2/src/test/java/org/apache/commons/AbstractVfsTestCase.java

```diff
@@ -136,7 +118,7 @@ public static File getTestDirectory(final String name) {
     /**
      * Makes a file canonical
      */
-    public static File getCanonicalFile(final File file) {
+    private static File getCanonicalFile(final File file) {
```

Comment:
There is no need to hide methods in tests, it just makes reusing code more difficult when using discovery features in IDEs like CTRL-space for method names.

Reply:
I find it makes it easier to detect unused code along with all the other typical reasons to default to private. That said I don't feel strongly about this so I'll just revert.